server/protocols/webadmin/webadmin-data/pom.xml

```diff
@@ -34,7 +34,28 @@
     <dependencies>
         <dependency>
             <groupId>${james.groupId}</groupId>
-            <artifactId>event-sourcing-event-store-memory</artifactId>
+            <artifactId>apache-james-mailbox-api</artifactId>
```

Comment:
Why do I need mailbox related stuff in data projects? 

Looks like the EML route should be in webadmin-mailbox...